places/views.py

```diff
@@ -1,3 +1,66 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response import JsonResponse
+from django.views         import View
+
+from places.models        import Place
+
+class PlaceListView(View):    
+    def get(self, request):
+        try:
+            filter_prefixes = {
+                'menu'     : 'category__menu_id',
+                'category' : 'category_id'
+            }
+            
+            filter_set = {filter_prefixes.get(key) : value[0] for (key, value) in dict(request.GET).items()}
```

Comment:
value[0]을 붙이는 이유가 뭔가요??

```suggestion
            filter_set = {filter_prefixes.get(key) : value for (key, value) in dict(request.GET).items()}
```

Reply:
result결과값이 배열형태라서 uri를 category=1로 요청하였을때 응답으로 category_id에 대한 키값을 ['1']와 같이 배열에 담아서 보내게 되어 데이터를 못읽어옵니다. 그래서 value[0]을 주게되면 키값을 1로 받아오게 되므로 정상적으로 데이터가 나옵니다. 
사실 승현멘토분께서 풀어주신거라 제가 이해한 부분이 맞는지....모르겠습니다ㅠ

Reply:
주현님! client에서 요청 보내는 형태가 `:8000?category=[1,2]`와 같은 방식으로 배열에 담아서 보내주고 있나요?

Reply:
네네 category=[1] 형태로 담아서 보내주기 때문에 value[0]으로 해야 category=1으로 보내주게 됩니다.

Reply:
그렇다면 아래와 같이 __in을 사용하시면 될 것 같습니다.
```python
            filter_prefixes = {
                'menu'     : 'category__menu_id',
                'category' : 'category_id__in'
            }

filter_set = {filter_prefixes.get(key) : value for (key, value) in dict(request.GET).items()}
```